src/inmanta/server/services/paramservice.py

```diff
@@ -200,54 +217,69 @@ async def _update_param(
 
         return recompile and value_updated
 
-    @handle(methods.set_param, param_id="id", env="tid")
+    @handle(methods.set_param, name="id", env="tid")
     async def set_param(
         self,
         env: data.Environment,
-        param_id: str,
+        name: str,
         source: ParameterSource,
         value: str,
-        resource_id: str,
+        resource_id: Optional[str],
         metadata: JsonType,
         recompile: bool,
+        expires: bool = False,
```

Comment:
I don't get this.

* This expires field is only present on the `@handle` method and not on the `@method`. Why?
* Why did we default the `expires` field to `False`? As far as I can see this method allow a user to set parameters and fact. This default doesn't make sense for acts, right?